app/build.gradle

```diff
@@ -131,40 +131,41 @@ repositories {
 
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
-    compile 'com.android.support:appcompat-v7:25.1.0'
-    compile 'com.android.support:cardview-v7:25.1.0'
-    compile 'com.android.support:design:25.1.0'
+    compile "com.android.support:appcompat-v7:$supportLibVersion"
+    compile "com.android.support:cardview-v7:$supportLibVersion"
+    compile "com.android.support:design:$supportLibVersion"
     compile 'com.android.support:multidex:1.0.1'
-    compile 'com.android.support:recyclerview-v7:25.1.0'
-    compile 'com.android.support:support-annotations:25.1.0'
+    compile "com.android.support:recyclerview-v7:$supportLibVersion"
+    compile "com.android.support:support-annotations:$supportLibVersion"
     compile 'com.facebook.android:facebook-android-sdk:4.7.0'
-    compile 'com.github.frankiesardo:auto-parcel:0.3.1'
-    annotationProcessor 'com.github.frankiesardo:auto-parcel-processor:0.3.1'
-    compile 'com.google.android.gms:play-services-gcm:10.0.1'
-    compile 'com.google.android.gms:play-services-wallet:10.0.1'
+    compile "com.github.frankiesardo:auto-parcel:$autoparcelVersion"
+    annotationProcessor "com.github.frankiesardo:auto-parcel-processor:$autoparcelVersion"
+    compile "com.google.android.gms:play-services-gcm:$playservicesVersion"
+    compile "com.google.android.gms:play-services-wallet:$playservicesVersion"
     compile 'com.google.android.exoplayer:exoplayer:r1.5.5'
-    compile 'com.google.dagger:dagger:2.0'
-    annotationProcessor 'com.google.dagger:dagger-compiler:2.0'
+    compile "com.google.dagger:dagger:$daggerVersion"
+    annotationProcessor "com.google.dagger:dagger-compiler:$daggerVersion"
     compile 'com.jakewharton:butterknife:7.0.1'
     compile 'com.jakewharton:process-phoenix:1.0.2'
     compile 'com.jakewharton.timber:timber:3.0.1'
     compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     compile 'com.stripe:stripe-android:1.0.3'
-    debugCompile 'com.squareup.leakcanary:leakcanary-android:1.3.1'
-    releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.3.1'
-    compile 'com.squareup.okhttp3:logging-interceptor:3.1.2'
-    compile 'com.squareup.okhttp3:okhttp-urlconnection:3.1.2'
+    debugCompile "com.squareup.leakcanary:leakcanary-android:$leakCanaryVersion"
+    releaseCompile "com.squareup.leakcanary:leakcanary-android-no-op:$leakCanaryVersion"
+    compile "com.squareup.okhttp3:okhttp:$okhttp3Version"
```

Comment:
Both `logging-interceptor` and `okhttp-urlconnection` depend on `okhttp`, it seems odd not to list it explicitly.